<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personnel</class>
 <widget class="QDialog" name="personnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>731</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajout</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Ajouter_personnel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>331</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter personnel</string>
     </property>
     <widget class="QLineEdit" name="id_perso">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_perso">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="idposte">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="telephone">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajoutperso">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>idposte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Affichage</string>
    </attribute>
    <widget class="QPushButton" name="imprbutt">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdfbutt">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>721</width>
       <height>371</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_perso">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>691</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>0</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>trier..</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nomperso</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>idposte</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pdfbutt_2">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="imprbutt_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <widget class="QLineEdit" name="id_a_supp">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="supp_perso">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>160</y>
         <width>172</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLineEdit" name="searchbar">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Search ID</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabperso">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>541</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="selectidperso">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>FIND</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TELEPHONE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID-POSTE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="newname">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="newprename">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="newphone">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="newemail">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="newidp">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="miseajour">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>280</y>
         <width>172</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mise A Jour</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>-29</y>
         <width>771</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>tableWidget</zorder>
      <zorder>searchbar</zorder>
      <zorder>label_12</zorder>
      <zorder>tabperso</zorder>
      <zorder>selectidperso</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>newname</zorder>
      <zorder>newprename</zorder>
      <zorder>newphone</zorder>
      <zorder>newemail</zorder>
      <zorder>newidp</zorder>
      <zorder>miseajour</zorder>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>118</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
